src/sgwu/context.c

```diff
@@ -250,3 +254,220 @@ sgwu_sess_t *sgwu_sess_add_by_message(ogs_pfcp_message_t *message)
 
     return sess;
 }
+
+void sgwu_sess_urr_acc_add(sgwu_sess_t *sess, ogs_pfcp_urr_t *urr, size_t size, bool is_uplink)
```

Comment:
It's called is_uplink here but is_ul elsewhere, would this have anything to do with it?